api/v1alpha2/linodeobjectstoragebucket_types.go

```diff
@@ -37,16 +44,22 @@ type LinodeObjectStorageBucketSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	Region string `json:"region"`
 
+	// ACL sets The Access Control Level of the bucket using a canned ACL string
+	// +optional
+	// +kubebuilder:default=private
+	// +kubebuilder:validation:Enum=private;public-read;authenticated-read;public-read-write
+	ACL ObjectStorageACL `json:"acl,omitempty"`
+
+	// corsEnabled enables for all origins in the bucket .If set to false, CORS is disabled for all origins in the bucket
+	// +optional
+	// +kubebuilder:default=true
+	CorsEnabled *bool `json:"corsEnabled,omitempty"`
```

Comment:
Does it need to be a pointer?

Reply:
Ah, I see. I think we don't need to make ours a pointer. We can just pass the pointer to the real value when doing create/update opts.